### PR TITLE
Integrate cPRO/LTT/inform

### PR DESCRIPTION
**Pull request by ivan-c**
- Add Keycloak OIDC client config for cPRO OpAuth integration (clientid `ltt_cpro`)
- Adapt docker-compose config from dhair2

See [PT#186506040](https://www.pivotaltracker.com/story/show/186506040)

### CODE REVIEW

**Comment by mcjustin** [comment]
Just adding a link to https://www.pivotaltracker.com/story/show/186506040

File: dev/cpro.env.default
Comment (mcjustin) [inline dev/cpro.env.default:1]: @ivan-c  will this file be managed on the server by puppet or this environment, or should I plan to bring it over manually?

**Review by mcjustin — APPROVED** [review]
I don't follow how some of this fits together, but trust that you have a vision here :)

File: dev/cpro.env.default
Comment (ivan-c) [reply dev/cpro.env.default:1]: we'll need to manually manage the per-service config files for now, but it shouldn't be difficult for puppet (or ansible :wink:) to manage them

I need to hear more about the existing LTT deploy, but we also may need to migrate it to a different VM (running traefik)

**Pull request merged**